scripts/pylint_all.py

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env python3
+
+"""
+Performs pylint on all python files in the project repo's test directory recursively.
+
+This script is meant to be run from the CI but can also be easily in local dev environment,
+where you can optionally pass `-d` as command line argument to let this script abort on first error.
+"""
+
+from os import path, walk, system
+from sys import argv, exit as brexit
```

Comment:
To be honest, I'd vote for remain, I mean, for rename :wink:

Reply:
Is aexit besser?